sdk/containerregistry/azure-containers-containerregistry/README.md

```diff
@@ -256,23 +256,7 @@ try {
 
 ## Troubleshooting
 
-All container registry service operations will throw a
-[HttpResponseException][HttpResponseException] on failure.
-
-```java readme-sample-getProperties
-DefaultAzureCredential credential = new DefaultAzureCredentialBuilder().build();
-ContainerRepository containerRepository = new ContainerRegistryClientBuilder()
-    .endpoint(endpoint)
-    .audience(ContainerRegistryAudience.AZURE_RESOURCE_MANAGER_PUBLIC_CLOUD)
-    .credential(credential)
-    .buildClient()
-    .getRepository(repositoryName);
-try {
-    containerRepository.getProperties();
-} catch (HttpResponseException exception) {
-    // Do something with the exception.
-}
-```
+See our [troubleshooting guide](https://github.com/Azure/azure-sdk-for-java/blob/0eb74418dc7a5ca2e40f954b3d7ce865321b6d86/sdk/containerregistry/azure-containers-containerregistry/TROUBLESHOOTING.md) for details on how to diagnose various failure scenarios.
```

Comment:
Probably don't want this pointing to a specific SHA 😄

Reply:
I didn't but I had to :(
- there is no such file on main
- we have a link verification step that [really wants](https://dev.azure.com/azure-sdk/public/_build/results?buildId=1666623&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=c9190431-7417-53a5-3fc1-7c7e026cd79d) me to have an absolute link here